src/Payments.Mvc/ClientApp/components/multiCustomerControl.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import { InvoiceCustomer } from '../models/InvoiceCustomer';
+import * as ArrayUtils from '../utils/array.js'; 
+
+const emailRegex = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/;
+
+interface IProps {
+    customers: InvoiceCustomer[];
+    onChange: (value: InvoiceCustomer[]) => void;
+}
+
+interface IState {
+    hasMultipleCustomers: boolean;
+    multiCustomerInput: string;
+}
+
+export default class DiscountInput extends React.Component<IProps, IState> {
+
+    constructor(props) {
+        super(props);
+        const { customers } = props;
+
+        this.state = {
+            hasMultipleCustomers: customers && customers.length,
+            multiCustomerInput: "",
+        }
+    }
+
+    public render() {
+        const { customers, onChange } = this.props;
+        const { hasMultipleCustomers, multiCustomerInput } = this.state;
+
+        if (!hasMultipleCustomers) {
+            const customer = customers.length ? customers[0] : {};
+            return (
+                <div className="">
+                    <h2>Customer Info</h2>
+                    <div className="form-group">
+                        <label>Customer Email</label>
+                        <div className="input-group">
+                            <input
+                                type="email"
+                                className="form-control"
+                                placeholder="johndoe@example.com"
+                                onChange={(e) => { onChange([{ email: e.target.value }]) }}
+                                value={customer.email}
+                            />
+                            <div className="input-group-append">
+                                <button className="btn btn-outline-secondary" type="button" onClick={this.enableMultiCustomer}>
+                                    <i className="fa fa-plus" />
```

Comment:
@jpknoll I'd remove the + icon, or at least add a space between the 2 icons

Reply:
I'm suspending design till a later PR